### ISSUE TITLE
Webhooks not firing

### ISSUE THREAD

**Issue by sotirisf** [umbraco/Umbraco.Cloud.Issues#596]
### Issue description

Webhooks on Umbraco Cloud have suddenly stopped working. I have set up webhooks on multiple environments and projectds with Zapier, but Zapier does not receive anything. I also tried with ﻿https://webhook.site/ and got the same results (nothing came through).

Tested with making dummy changes on my local and pushing to master - made sure that deployment was finished correctly.

**Comment by RyuLindow** [comment]
Hey @sotirisf 

I've tested it with @Aaen and nothing fires either.

Are you able to check when you've received the last webhook from Cloud?

I'm trying to determine whether it was before we migrated to the new infrastructure or if it was one of the platform changes we've made.

There's also an option that it could have something to do with site extensions but this is just a complete guess.

**Comment by sotirisf** [comment]
Last notification I have is on Aug 25 as far as I can see.

**Comment by RyuLindow** [comment]
Hmm, it could be related to [https://status.umbraco.io/incidents/vx3cplv505mg](https://status.umbraco.io/incidents/vx3cplv505mg)

I'll check with @sajumb about it :)

**Comment by sajumb** [comment]
Thanks for the report. We are looking into this!

**Comment by sajumb** [comment]
We will have a fix out either today or early next week. We are sorry for the inconvenience the downtime may have caused you.

**Comment by sotirisf** [comment]
I'm not sure if any actions have been taken yet, but in any case no change has been observed so far.

**Comment by sajumb** [comment]
Yes, sorry about this taking a bit longer than expected. The fix is delayed a couple of days and is now expected to be released on Friday.

**Comment by sajumb** [comment]
Sorry for the wait. The good news is that the webhooks functionality is working again.

**Comment by sotirisf** [comment]
Still not getting anything. We've got multiple active projects, and pushing multiple times per day to environments with webhooks, but no joy.

**Comment by sajumb** [comment]
My bad. The cloud team stated the issue was fixed and they are normally always right. Our support, However, like you, our support has indicated that there is still one last outstanding issue that we just need to fix. Sorry again...

**Comment by RyuLindow** [comment]
Heya @sotirisf 

The feature is working again. I have verified it on a test project and got the expected webhook payload :)

Can you please, check on your end?
